mlflow/utils/validation.py

```diff
@@ -52,14 +64,32 @@ def _validate_metric(key, value, timestamp):
             "double (64-bit floating point)" % (value, key, timestamp),
             INVALID_PARAMETER_VALUE)
 
-    if not isinstance(timestamp, numbers.Number) or timestamp < 0 or \
-            timestamp < np.iinfo(np.int64).min:
+    if not isinstance(timestamp, numbers.Number) or timestamp < 0:
```

Comment:
Small change, removed a redundant check here that @dbczumar caught while reviewing #955